setup/module/api/db.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_dynamodb_table" "tags" {
   name           = "l0-${var.name}-tags"
-  read_capacity  = 50
+  read_capacity  = 5
```

Comment:
Do we know the cost, in terms of time and latency, as to when autoscaling takes effect? I can see the argument for a lower initial value so that autoscaling gets to do the heavy lifting (and the potential cost savings therein), but we may want to consider keeping this value at `50` if it was deemed to be reasonable to begin with.

Reply:
Here's an image of read capacity for the tags table. The deploy I used was a 5-min sleep, and the three spike correspond to three different attempts at hammering the api.
1. `for x in {1..100} ; do nohup l0 task create test-env task-$x test-dpl:latest & done`
2. `for x in {1..200} ; do nohup l0 task create test-env task-$x test-dpl:latest & done`
3. `for y in {1..10} ; do for x in {1..100} ; do nohup l0 task create test-env task-$x test-dpl:latest & done ; sleep 20 ; done`

<img width="907" alt="Screen Shot 2019-04-10 at 10 33 18 AM" src="https://user-images.githubusercontent.com/5035800/55900665-895a9c80-5b7c-11e9-95c8-b2cddf08d58b.png">

Reply:
@Cryowatt have you observed were the read capacity is tending to average when autoscaling is enabled?

Reply:
Most of our services are duck-curved, so the capacity trends with when our customers are awake. It seems that during low hours we're down at 5 read units (probably a good minimum for auto-scaling). During busy hours our dev environment hits about 200, but tends to move up and down a lot. I think setting the min low and letting the autoscaling handle things should be fine.

Reply:
Thanks @Cryowatt. Your comments suggest that we should up our `max_capacity` read units count to over 200 as well.